cmd/cli/cmd/cluster/create/kind.go

```diff
@@ -0,0 +1,29 @@
+package create
+
+import (
+	"time"
+
+	"capact.io/capact/internal/cli/cluster/create"
+	"github.com/spf13/cobra"
+)
+
+func NewKind() *cobra.Command {
+	var opts create.KindOptions
+
+	cmd := &cobra.Command{
+		Use:   "kind",
+		Short: "Provision local kind cluster",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return create.Kind(cmd.Context(), opts)
+		},
+	}
+	cmd.Flags().StringVar(&opts.Name, "name", "kind-dev-capact", "cluster name, overrides config")
+	cmd.Flags().StringVar(&opts.Config, "cluster-config", "", "path to a kind config file")
+	cmd.Flags().StringVar(&opts.ImageName, "image", "", "node docker image to use for booting the cluster")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&opts.ImageName, "image", "kindest/node:v1.19.1", "node docker image to use for booting the cluster")
```

Currently we use the `kindest/node:v1.19.1` and now cluster installs `v1.21.1`. This can be problematic, as some resources were deprecated and removed after 1.19, so u may run into an issue in next task


BTW not sure if this is a good place, probably we should have it close to our default config :)